### ISSUE TITLE
Paper

### ISSUE THREAD

**Issue by AliasChenYi** [open-mmlab/mmpose#3000]
### What is the feature?

Hello, I would like to ask if I can publish a paper after modifying your official source code.

### Any other context?

Hello, I would like to ask if I can publish a paper after modifying your official source code.

**Comment by fducau** [comment]
Yes, check the license (Apache 2.0) https://github.com/open-mmlab/mmpose/blob/main/LICENSE

Also check the [Citation](https://github.com/open-mmlab/mmpose?tab=readme-ov-file#citation) bit in the readme.
```
@misc{mmpose2020,
    title={OpenMMLab Pose Estimation Toolbox and Benchmark},
    author={MMPose Contributors},
    howpublished = {\url{https://github.com/open-mmlab/mmpose}},
    year={2020}
}
```
